subprojects/dependency-management/src/main/java/org/gradle/api/internal/artifacts/transform/TransformationSubject.java

```diff
@@ -46,6 +49,10 @@ public static TransformationSubject initial(ComponentArtifactIdentifier artifact
      */
     public abstract ImmutableList<File> getFiles();
 
+    public abstract String getInitialFileName();
```

Comment:
This sounds like a leaky abstraction here. Why do we need the initial file name?

Reply:
I think what we should capture here is the _role_ or _intent,_ not the actual thing. So if we want to keep this, let's rename it to `getHumanReadableIdentifier()` and add whatever we can synthesize from that discussion in the Javadoc.

Reply:
`humanReadableIdentifier` it is.